spec/models/user_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  it { should validate_presence_of(:title) }
+  it { should validate_length_of(:title).is_at_most(250) }
+  it { should validate_numericality_of(:comments_counter).only_integer.is_greater_than_or_equal_to(0) }
+  it { should validate_numericality_of(:likes_counter).only_integer.is_greater_than_or_equal_to(0) }
+end
```

Comment:
- You have done great so far :+1: But there are a few requirements you still need to meet before moving to the next milestone. :+1: :100: 

- [ ] You have done great adding validations to your models. but Looking at the screenshot below, you can see that your tests are failing. This is the test file for the `User` model but you seem to have the same tests you wrote for the Post model here. I'll suggest you actually set up your test suite correctly and write actual tests for the validation in the users model. ALso you are meant to write tests for all the methods in each model also. in this case, In the users model, you need to also test `three_most_recent_post` which is a method that's meant to be in the users model. 

here's an example and a guideline of what your `user_spec` could look like.
```ruby
RSpec.describe User, type: :model do
  subject { User.new(name: 'Lilly', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Poland.', posts_counter: 0) }

  before { subject.save }

  it 'name should be present' do
    subject.name = nil
    expect(subject).to_not be_valid
  end
```  

- - Failed tests screenshot reference:
 
![Screenshot from 2023-10-07 20-01-22](https://github.com/geekelo/rails-blog_app/assets/107632780/0c0f6259-8c6e-454c-87ce-5328e7a27fc3)